technical-embedded-ui/tpp-ui/src/app/initial-request/initial-request.component.ts

```diff
@@ -54,6 +57,15 @@ export class InitialRequestComponent implements OnInit {
           this.serviceSessionId.setValue(params.serviceSessionId);
         }
       });
+
+    this.getIPAddress();
+  }
+
+  getIPAddress() {
+    this.client.get('http://api.ipify.org/?format=json').subscribe(
```

Comment:
I think hardcoded 1.1.1.1 is perfectly fine, external service is not a very safe solution

Reply:
ok